BMM.Core/Models/PO/BasePO.cs

```diff
@@ -0,0 +1,8 @@
+using MvvmCross.ViewModels;
+
+namespace BMM.Core.Models.PO
+{
+    public abstract class BasePO : MvxNotifyPropertyChanged
```

Comment:
what does PO stand for?

Reply:
Presentation Object, it is basically model of object which interacts with view and it not View Model, thats why it implements INotifyPropertyChanged. It is used e.g. for cells or for custom views.

Reply:
please add a summary comment to BasePO explaining it.